<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fridgeexplorer</class>
 <widget class="QMainWindow" name="Fridgeexplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fridgeexplorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="button_add">
      <property name="text">
       <string>add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_time">
      <property name="text">
       <string>DoW HH:MM:SS</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2">
     <widget class="QPushButton" name="button_remove">
      <property name="text">
       <string>remove</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="5">
     <widget class="QGroupBox" name="gb_menu">
      <property name="title">
       <string>Menu Suggestion</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_menu_quick">
         <property name="text">
          <string>be quick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_menu_cook">
         <property name="text">
          <string>let's cook</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_menu_mobile">
         <property name="text">
          <string>take-away</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionconnect_database"/>
    <addaction name="actionmanage_recipes"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionconnect_database">
   <property name="text">
    <string>connect database ...</string>
   </property>
  </action>
  <action name="actionmanage_recipes">
   <property name="text">
    <string>manage recipes</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
